core/src/test/kotlin/io/sharing/server/core/product/domain/ProductTest.kt

```diff
@@ -50,6 +53,52 @@ class ProductTest {
             "image6", "image7", "image8", "image9", "image10"
         )
     }
+
+    @Test
+    fun `등록된 상품을 승인을 할 수 있다`() {
+        val product = createProduct().apply {
+            this.status = ProductStatus.REGISTERED
+        }
+
+        product.approve()
+
+        assertThat(product.status).isEqualTo(ProductStatus.AVAILABLE)
+    }
+
+    @EnumSource(value = ProductStatus::class, names = ["REGISTERED"], mode = EXCLUDE)
+    @ParameterizedTest
+    fun `등록 상태가 아니면 상품 승인 할 수 없다`(status: ProductStatus) {
```

Comment:
저런 형태 포맷으로 작성해주면 일관성있는 이름으로 작성할 수 있게 되어 가독성으 높일 수 있습니다.

```suggestion
    fun `상품 승인 실패 - 등록 상태가 아닌 경우`(status: ProductStatus) {
```